src/Microsoft.Azure.SignalR.Common/Endpoints/AadAccessKey.cs

```diff
@@ -41,13 +68,43 @@ public async Task AuthorizeAsync(string endpoint, int? port, string serverId, Ca
             return await base.GenerateAccessToken(audience, claims, lifetime, algorithm);
         }
 
-        public Task<string> GenerateAadToken()
+        public async Task UpdateAccessKeyAsync(IServerNameProvider provider, ServiceEndpoint endpoint, ILoggerFactory loggerFactory)
```

Comment:
So what if the caller `UpdateAccessKeyAsync(endpoint1)` then `UpdateAccessKeyAsync(endpoint2)`, `endpoint2` never get authorized?

Reply:
They don't need if they shared the same `AccessKey` instance.